JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,40 +1,60 @@
 //
 //  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
+//  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
 //
 
 import Foundation
 
 // 과일 저장소 타입
 final class FruitStore {
-    static let defaultStock: UInt = 10
     private(set) var stockByFruit = [Fruit: UInt]()
-    
+
     enum Fruit: CaseIterable {
         case strawberry, banana, pineapple, kiwi, mango
+
+        var sequence: Int {
+            switch self {
+            case .strawberry:
+                return 0
+            case .banana:
+                return 1
+            case .pineapple:
+                return 2
+            case .kiwi:
+                return 3
+            case .mango:
+                return 4
+            }
+        }
     }
-    
-    init(stock: UInt = defaultStock) {
+
+    init(stock: UInt = 10) {
         for fruit in Fruit.allCases {
             stockByFruit[fruit] = stock
         }
     }
-    
+
     func subtractStock(fruit: Fruit, count: UInt) throws {
-        guard let fruitStock = stockByFruit[fruit] else {
-            throw JuiceMakerError.notExistFruit
-        }
+        let fruitStock = stock(byFruit: fruit)
         guard fruitStock >= count else {
             throw JuiceMakerError.outOfStock
         }
         stockByFruit[fruit] = fruitStock - count
     }
-    
+
     func addStock(fruit: Fruit, count: UInt) throws {
-        guard let fruitStock = stockByFruit[fruit] else {
-            throw JuiceMakerError.notExistFruit
+        self.stockByFruit[fruit] = stock(byFruit: fruit) + count
+    }
+    
+    func stock(byFruit: Fruit) -> UInt {
```

Comment:
파라미터를 다음과 같이 바꾸면 깔끔할 것 같아요 😊

```suggestion
    func stock(byFruit fruit: Fruit) -> UInt {
```